t/Util.pm

```diff
@@ -698,4 +699,36 @@ sub get_tracer {
     return $read_trace;
 }
 
+sub run_picotls_client {
+    my($opts) = @_;
+    my $port = $opts->{port}; # required
+    my $host = $opts->{host} // '127.0.0.1';
+    my $path = $opts->{path} // '/';
+    my $cli_opts = $opts->{opts} // '';
+    my $connection = $opts->{keep_alive} ? "keep-alive" : "close";
+
+    my $cli = bindir() . "/picotls/cli";
+    die "picotls-cli ($cli) not found" unless -e $cli;
+
+    my $tempdir = tempdir();
+    my $cmd = "exec $cli $cli_opts $host $port > $tempdir/resp.txt 2>&1";
+    diag $cmd;
+    open my $fh, "|-", $cmd
+        or die "failed to invoke command:$cmd:$!";
+    autoflush $fh 1;
+    print $fh <<"EOT";
+GET $path HTTP/1.1\r
+Host: $host:$port\r
+Connection: $connection\r
+\r
+EOT
+    Time::HiRes::sleep(0.1) until -e "$tempdir/resp.txt" && -s "$tempdir/resp.txt" > 0;
```

Comment:
I'm not sure if this change (from `sleep 1`) is worth the complexity.

If the intent is to do proper, we should keep the write-side open, until the read side is closed (i.e. below `do { ... <$fh> };`). To prevent hangup when something goes wrong, we also need to employ a timer.

But the question is if those changes are worth the complexity. If not, a simple `sleep 1` as we've had in the past would be enough.